app/_hub/reedelk/reedelk-transformer/index.md

```diff
@@ -0,0 +1,205 @@
+  # Defaults to the url setting in your publisher profile.
+
+source_url: https://github.com/reedelk/kong-plugin-reedelk-transformer
+
+license_type: Apache-2.0
+  # (Optional) For open source, use the abbreviations in parentheses at:
+  # https://opensource.org/licenses/alphabetical
+
+license_url: https://github.com/reedelk/kong-plugin-reedelk-transformer/blob/master/LICENSE.txt
+
+kong_version_compatibility:
+  community_edition:
+    compatible:
+      - 2.1.x
+      - 1.5.x
+  # enterprise_edition:
+  #  compatible:
+  #    - 2.1.x
+  #    - 1.5.x
+
+params:
+  name: reedelk-transformer
+  api_id: false
+  service_id: true
+  consumer_id: false
+  route_id: true
+  protocols: ["http", "https"]
+  dbless_compatible: yes
+  dbless_explanation:
+
+  config:
+    - name: upstream_transformer_url
+      required: no
+      default:
+      value_in_examples: http://myhost/upstream/transform
+      description:
+        The URL of the Reedelk REST flow endpoint to invoke for the Upstream
+        request transformation.
+
+    - name: downstream_transformer_url
+      required: no
+      default:
+      value_in_examples: http://myhost/downstream/transform
+      description:
+        The URL of the Reedelk REST flow endpoint to invoke for the Downstream
+        request transformation.
+
+  extra:
+    The `upstream_transformer_url` and `downstream_transformer_url` are the URLs
+    of the Reedelk REST flow endpoint to invoke for the Upstream/Downstream
+    request/response transformations; e.g., `http://localhost:8888/apiabledev/transform`.
+###############################################################################
+# END YAML DATA
+
+###############################################################################
+# BEGIN MARKDOWN CONTENT
+---
+
+## Installation
+
+### Prerequisites
+
+To use the Reedelk plugin for Kong, you must first install the Reedelk IntelliJ
+IDEA flow designer plugin using either of the following options:
+
+- **IntelliJ Marketplace**: From **IntelliJ Preferences > Plugin > Marketplace**,
+  search for `Reedelk`, install the plugin, and restart IntelliJ.
+
+- **Manual install**: From **IntelliJ Preferences > Plugin > Settings icon >
+  Install Plugin From Disk**, and restart IntelliJ.
+
+### Install Kong Plugin Steps
+
+1. Clone the `kong-plugin-reedelk-transformer`:
+
+   ```bash
+      git clone https://github.com/reedelk/kong-plugin-reedelk-transformer.git
+      cd kong-plugin-reedelk-transformer
+   ```
+
+2. Install the module `kong-plugin-reedelk-transformer`:
+
+   ```bash
+   luarocks install kong-plugin-reedelk-transformer-0.1.0-1.all.rock
+   ```
+
+3. Add the custom plugin to `kong.conf`:
+
+   ```
+   plugins = ...,reedelk-transformer
+   ```
+
+4. [Restart](/2.1.x/cli/#kong-restart) Kong.
+
+### Building and Packaging the Kong Plugin
+
+1. The [LuaRocks](http://luarocks.org) package manager must be
+   [installed](https://github.com/luarocks/luarocks/wiki/Download).
```

Comment:
This is more of a prereq than a step.

Reply:
I know, I thought about that. This is how Reedelk has it on their github doc page.  I don't want to have a Prereq subtitle though to be consistent with the rest of the Prereq headings since it is only one thing and the head levels are already getting deeper.